tools/roxvet/analyzers/testtags/analyzer.go

```diff
@@ -0,0 +1,80 @@
+package testtags
+
+import (
+	"go/ast"
+	"path/filepath"
+	"strings"
+
+	"github.com/stackrox/rox/tools/roxvet/common"
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+const doc = `Ensure that our *_test.go files include a //go:build <TAG>, otherwise they won't be run.'`
```

Comment:
Could you include instructions how to rectify this if the linter find such a file?
E.g. "The tag can be added by running prepend-go-build-to-tests.sh" or something comparable?

Reply:
Sure, good idea